packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-propagation.spec.js

```diff
@@ -161,7 +161,9 @@ function doEventsBubbleToDocFrag() {
 describe('event propagation in disconnected tree', () => {
     it('propagate event from a child element in a document fragment', () => {
         const fragment = document.createDocumentFragment();
-        const nodes = createShadowTree(fragment);
+        const nodes = createShadowTree(document.body);
+        // Add to body and then move to fragment because lwc element is not hydrated until it is connected to the document
+        fragment.appendChild(nodes['x-shadow-tree']);
```

Comment:
This looks like an inconsistent behavior with the previous mechanism. I don't fully understand the comment, could you provide more details?